randovania/games/prime2/json_data/Agon Wastes.txt

```diff
@@ -1492,9 +1498,13 @@ Extra - in_dark_aether: False
           All of the following:
               Space Jump Boots
               Any of the following:
-                  After Central Mining Station Pirates
-                  Scan Visor and Combat/Scan Dash (Advanced)
+                  After Central Mining Station Pirates or Use Screw Attack (Space Jump)
```

Comment:
You removed the regular Scan Dash path, so you need to re-add here.

Reply:
Not sure how this happened, but it's back now haha.